File: Unidad 5 Java/Programa9U5.java
package com.mycompany.programa9u5;
import java.util.Scanner;

public class Programa9U5 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese el sueldo diario del trabajador: $");
        double sueldoDiario = sc.nextDouble();

        System.out.print("Ingrese el tiempo laborado (en días): ");
        int tiempoLaborado = sc.nextInt();

        double finiquito = calcularFiniquito(sueldoDiario, tiempoLaborado);

        System.out.println("Finiquito del trabajador: $" + finiquito);
    }

    static double calcularFiniquito(double sueldoDiario, int tiempoLaborado) {
        return sueldoDiario * tiempoLaborado;
    }
}

File: Unidad 5 C#/Programa9U5.cs
using System;

namespace programa9u5
{
    internal class Programa9U5
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese el sueldo diario del trabajador: $");
            double sueldoDiario = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el tiempo laborado (en días): ");
            int tiempoLaborado = Convert.ToInt32(Console.ReadLine());

            double finiquito = CalcularFiniquito(sueldoDiario, tiempoLaborado);

            Console.WriteLine("Finiquito del trabajador: $" + finiquito);
        }

        static double CalcularFiniquito(double sueldoDiario, int tiempoLaborado)
        {
            return sueldoDiario * tiempoLaborado;
        }
    }
}